ansible/deploy-update.yml

```diff
@@ -12,4 +12,4 @@
   hosts: mailserver
   become: true
   roles:
-    - { role: mailcow, tags: update }
```

Comment:
I would actually prefer to have the updates tag here. This way we make sure it is not forgotten when running the yaml. But it's a matter of opinion.

Reply:
This tags all the tasks in mailcow role with the update tag, which is obviously not what we want to do.

> I would actually prefer to have the updates tag here. This way we make sure it is not forgotten when running the yaml. But it's a matter of opinion.

This is also my preference but I'm not sure if it's possible or I don't know how to do this. :P

Reply:
Resolved in: https://github.com/PrivacyLx/devops/pull/58/commits/81bfb8ac314fe40e693786f62f2a2f764a8908e3